cloud/user/user.go

```diff
@@ -130,27 +131,41 @@ func IsRoleValid(role string) error {
 	return ErrInvalidRole
 }
 
-func selectUser(users []astrocore.User) (astrocore.User, error) {
+func selectUser(users []astrocore.User, workspace bool) (astrocore.User, error) {
+	roleColumn := "ORGANIZATION ROLE"
+	if workspace {
+		roleColumn = "WORKSPACE ROLE"
+	}
```

Comment:
Do we need this header to be different for Org vs. workspace role when we print a list? What if the header was `ROLE` and the value has `ORGANIZATION_` or `WORKSPACE_` in it.

Reply:
I thought specifying the type of role would be nice. Especially since we are already changing the table in other ways.